src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/RequiredIntervalArgumentCollection.java

```diff
@@ -14,7 +14,7 @@
 public final class RequiredIntervalArgumentCollection extends IntervalArgumentCollection {
     private static final long serialVersionUID = 1L;
 
-    @Argument(fullName = "intervals", shortName = "L", doc = "One or more genomic intervals over which to operate", optional = false)
+    @Argument(fullName = "intervals", shortName = "L", suppressFileExpansion = true, doc = "One or more genomic intervals over which to operate", optional = false)
```

Comment:
Add comment here explaining why file expansion is suppressed for this arg. The comment should be a non-javadoc comment (eg., //) so that it won't be picked up by the doc generation code.

Reply:
done.